config/dev-local.js

```diff
@@ -29,6 +29,8 @@ module.exports = merge(base, devVm, {
 	},
 	server: {
 		graphqlUri: 'https://marketplace-api.dk1.kiva.org/graphql',
+		// Alternate graphql url (marketplace-api.dk1.kiva.org is redirected to this url)
+		// graphqlUri: 'https://gateway.development.kiva.org/graphql',
```

Comment:
It looks like you can remove these two lines and update line 31 with the new URI.

Reply:
Updated for app and server.